examples/ecs/relationships.rs

```diff
@@ -0,0 +1,192 @@
+//! Entities generally don't exist in isolation. Instead, they are related to other entities in various ways.
+//! While Bevy comes with a built-in [`Parent`]/[`Children`] relationship
+//! (which enables transform and visibility propagation),
+//! you can define your own relationships using components.
+//!
+//! Every relation has two sides: the source and the target.
```

Comment:
Teeny-weeny nit: this is the only instance of "relation" in this PR, it's "relationship" everywhere else
```suggestion
//! Every relationship has two sides: the source and the target.
```